mirar/processors/utils/image_plotter.py

```diff
@@ -0,0 +1,57 @@
+"""
+Module to plot images
```

Comment:
Please be slightly more explicit somewhere about what the plotting does. Are you basically just plotting the 2D image as a PNG? How do the regions get used?

Reply:
done, regions don't get used in the processor at all.